.release-please-manifest.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This is needed as starting point to create next versions. [Description](https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#initial-version)